tools/cockpit.spec

```diff
@@ -588,24 +565,6 @@ These files are not required for running Cockpit.
 %{_unitdir}/cockpit-session.socket
 %{_unitdir}/cockpit-session@.service
 
-%if %{build_pcp}
-
-%package -n cockpit-pcp
```

Comment:
Please add `Obsoletes: cockpit-pcp < 326` to cockpit-bridge.

https://docs.fedoraproject.org/en-US/packaging-guidelines/#renaming-or-replacing-existing-packages